strings/charoccurrence/charoccurrence_test.go

```diff
@@ -0,0 +1,40 @@
+package charoccurrence
+
+import (
+	"reflect"
+	"testing"
+)
+
+var testCases = []struct {
+	name     string       // test description
+	input    string       // user input
+	expected map[rune]int // expected return
```

Comment:
```suggestion
	name     string
	input    string
	expected map[rune]int
```
These are quite clear, there is no need for such redundant comments.

Reply:
- [x] fixed